packages/jobs/src/bins/rw-jobs-worker.ts

```diff
@@ -84,9 +84,10 @@ const setupSignals = ({
 }
 
 const main = async () => {
-  // TODO TOBBE For some reason the `parseArgs` type reports that it is only returning the single letter option flags as keys in this object (i, q, c, o), but it does return the alias names as well (id, queue, clear, workoff) I'd rather use the full alias names here as it's much easier to understand what the values are for.
-  // @ts-ignore
-  const { id, queue, clear, workoff } = parseArgs(process.argv)
+  const { id, queue, clear, workoff } = await parseArgs(process.argv)
```

Comment:
So currently nothing happens _internally_ with the return from `perform()`: if you schedule the job with `performLater()` then when the job succeeds, that's it. 

But, you can also run the job with `performNow()` or even call `perform()` itself. For that reason I wanted the user to be able to return whatever they want because they may want that result for something.

In a future release I wanted to have lifecycle hooks, like `beforePerform()` and `afterPerform()`. `afterPerform()` would accept as an argument whatever was returned from `perform()` so you could do something with it if you want.

So, it seemed like `any` was the best option here! Unless it's okay to make this `void` for the abstract function, and then when you write the real one in your own subclass you can have it return whatever you want? But if the default is `void` then will users assume their own should be `void` as well? How does the Typescript mind work? :)